### DISCUSSION TITLE
Contributing workflow

### DISCUSSION THREAD

**Discussion by paulaFenner** [paulaFenner/salesboard-app#10] [General]
Hey @Samuelmusic1 and @abaltazapinto 

I'm adding Scrimba's open source contributing file[ here.](https://github.com/ScrimbaBootcamp/progress-tracker/blob/main/CONTRIBUTING.md)
I think it would be a good idea for us to make changes on "new features" branches.

**Comment by Samuelmusic1** [comment]
This is really nice and will be helpful, thank you paula.
